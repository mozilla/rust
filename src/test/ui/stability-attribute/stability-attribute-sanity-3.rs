// More checks that stability attributes are used correctly

#![feature(staged_api)]

#![stable(feature = "stable_test_feature", since = "1.0.0")]

#[macro_export]
macro_rules! mac { //~ ERROR macro is missing a stability attribute
    () => ()
}

fn main() { }
