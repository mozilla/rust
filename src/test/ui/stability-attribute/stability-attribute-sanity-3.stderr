error: macro is missing a stability attribute
  --> $DIR/stability-attribute-sanity-3.rs:8:1
   |
LL | / macro_rules! mac {
LL | |     () => ()
LL | | }
   | |_^

error: aborting due to previous error

